packages/rsc/package.json

```diff
@@ -65,7 +65,7 @@
     "zod": "3.25.76"
   },
   "peerDependencies": {
-    "react": "^18 || ^19 || ^19.0.0-rc",
```

Comment:
I think it makes sense to drop support for 19-rc in ai v6. We can keep supporting it in v5 though